app/controllers/repos_controller.rb

```diff
@@ -51,6 +51,35 @@ def update
     end
   end
 
+  def remove
+    @repos = []
+    @repo = Repo.new(user_name: params[:user_name], name: name_from_params(params))
+
+    Repo.all.each do |x|
```

Comment:
This is going to be pretty expensive. Every time someone clicks that button we have to make 1,845 calls to the github API. I don't think we need a custom view to see all repos that you can remove, I would rather we only had a button on each page. We should probably store the push access data somewhere. Maybe a HABTM table that stores users that have push access that are also code-triage members. I don't know the best way to get this info, i'm thinking we should populate when a new user joins and maybe sync once a day when we do the issue sync.